runner.md

```diff
@@ -9,6 +9,8 @@ There is one runner per sprint.  The new runner is selected during the [Sprint P
 The runner's duties are as follows:
 * Each workday, check Honeybadger and [DataDog](https://app.datadoghq.com/logs?saved-view-id=2137895) for errors in RDSS applications.
 * Create a ticket for each error encountered, if one does not already exist.
-* Bring error tickets to the RDSS team's attention at check-in for prioritization. 
+* Bring error tickets to the RDSS team's attention at check-in for prioritization.
 * If a ticket is a work-stopping issue/considered high priority and is therefore assigned to the current sprint, it receives the "unplanned work" [label](work_cycles.md) in GitHub.
 * In the ticket, include a checkbox to mark the issue resolved in Honeybadger as part of the acceptance criteria if appropriate ([example here](https://github.com/pulibrary/pdc_describe/issues/1513)).
+
+* Check the failed jobs queue for PDC Describe in production. See [details](https://github.com/pulibrary/pdc_describe/issues/1574#issuecomment-1773374397)
```

Comment:
I would prefer to have your documentation in the docs directory. https://github.com/pulibrary/pdc_describe/pull/1582

```suggestion
* Check the failed jobs queue for PDC Describe in production. See [details](https://github.com/pulibrary/pdc_describe/blob/main/docs/sidekiq_jobs.md)
```